spyder/app/mainwindow.py

```diff
@@ -2420,34 +2420,53 @@ def render_issue(self, description='', traceback=''):
             "will be closed. Thanks! --->"
             ).format(__trouble_url__)
 
+        reminder_message_short = (
+            "<!--- PLEASE READ: Complete the following checklist. "
+            "Issues without it may be closed. --->"
+            )
+
+        bug_checklist = (
+            "* [ ] Searched issues page for similar reports\n"
+            "* [ ] Read and followed relevant sections of the"
+            "[Troubleshooting Guide]({0!s})\n"
+            "* [ ] Reproduced after updating (`conda update spyder`)\n"
+            "* [ ] Tried basic troubleshooting\n"
+            "    * [ ] Restarted Spyder\n"
+            "    * [ ] Ran `spyder --reset`\n"
+            "    * [ ] Reinstalled latest Anaconda\n"
+            "* [ ] Completed all applicable sections below\n"
+            ).format(__trouble_url_short__)
+
         # Make a description header in case no description is supplied
         if not description:
-            description = "### What steps will reproduce the problem?"
+            description = "### What steps reproduce the problem?"
 
-        # Make error section from traceback
+        # Make error section from traceback and add appropriate reminder header
         if traceback:
             error_section = ("### Traceback\n"
                              "```python-traceback\n"
                              "{}\n"
                              "```".format(traceback))
+            reminder_message = reminder_message_full
         else:
             error_section = ''
+            reminder_message = reminder_message_short + "\n\n" + bug_checklist
         issue_template = """\
 {reminder_message}
 
-## Problem Description
+## Description
 
 {description}
 
 {error_section}
 
-## Package Versions
+## Versions
 
-* Spyder version: {spyder_version} {commit}
-* Python version: {python_version}
-* Qt version: {qt_version}
-* {qt_api_name} version: {qt_api_version}
-* Operating system: {os_name} {os_version}
+* Spyder ver: {spyder_version} {commit}

```

Comment:
I think we have talked about this before. `ver` can be confusing and is not explicit enough for non-English speakers. So please revert this to `version`.

Reply:
I agree, explicit is better than implicit, and I don't often use abbreviations myself. See above for the reasons and alternatives.